ghc-9.2.5.stack.yaml

```diff
@@ -2,5 +2,5 @@ resolver: lts-20.3
 packages:
   - .
 extra-deps:
-  - daemons-0.3.0@sha256:356edaf06a8cf5e9f19685f9ce793ad6d2b6bce2e55379f6b2aff1b0c07a633f,3178
+  - hdaemonize-0.5.6
```

Comment:
I think `hdaemonize` is already in lts-20.3 so you don't need to make it an `extra-dep`.

Reply:
Thanks for the catch, Neil.